src/apps/schema/src/appRevamp/components/AddFieldModal/views/FieldForm.tsx

```diff
@@ -274,6 +310,8 @@ export const FieldForm = ({
       required: formData.required as boolean,
       settings: {
         list: formData.list as boolean,
+        limit: formData.limit as number,
```

Comment:
Have you tried creating a different field type that will cause these values to be undefined? I remember getting api errors on some field types that have undefined values before.

Reply:
On media type, there are no undefined values.